modules/apps/asset/asset-categories-admin-web/npmscripts.config.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
 			'<inputDir>/js/AssetCategoriesManagementToolbarDefaultEventHandler.es.js',
 			'<inputDir>/js/AssetCategoriesSelectorTag.es.js',
 		],
+		mode: 'incompatible',
```

Comment:
😱

Reply:
:joy: If you can think of a better name... However it's one of those temporary things we like... :roll_eyes:

Reply:
So, what was the rationale from renaming one temporary thing for another temporary thing?

Reply:
It's in the second commit.

Reply:
I'm not super excited about it, specially about the `incompatible` mode which seems closer to an error than a wanted feature. Surely you always prefer something compatible! 😂

Reply:
>  a new setting in npm-scripts to distinguish when to run webpack only, webpack + bundler, or only bundler

So can you spell this out for me here?

- `disabled` = only bundler
- `compatible` = webpack + bundler
- `incompatible` = webpack only

???

You say this is temporary, so what will the eventually final state be? (ie. what values and what will they mean?)

Reply:
See https://github.com/liferay/liferay-frontend-projects/pull/420#discussion_r582848009

`mode: 'default'` sounds like a good name.